packages/core/src/components/Modal/ModalFooter/ModalFooterButtons/ModalFooterButtons.tsx

```diff
@@ -9,7 +9,9 @@ import ModalFooter from "../ModalFooter";
 
 export interface ModalFooterButtonsProps extends VibeComponentProps {
   primaryButtonText: string;
-  secondaryButtonText: string;
+  secondaryButtonText?: string;
+  disablePrimaryButton?: boolean;
+  disableSecondaryButton?: boolean;
```

Comment:
We don't want to disable the secondary, just the primary, it's a UX consideration

Reply:
done